xdmod/install.sh

```diff
@@ -22,6 +22,9 @@ ARCHTYPE=`uname -m`
 dnf module -y reset nodejs
 dnf module -y install nodejs:16
 
+dnf -y module reset php
+dnf -y module install php:7.4
```

Comment:
I think this is the right version, but can't say for sure. I do know there are errors building the images currently.